### ISSUE TITLE
[Bug]: Cannot render the file Error when playing video on external player mpc-hc

### ISSUE THREAD

**Issue by 2400YrsOldLallo** [FreeTubeApp/FreeTube#4195]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

1. play video using mpc-hc
2. Get error which shows Cannot render the file from mpc-hc

### Expected Behavior

video plays without issues?

### Issue Labels

accessibility issue, content not loading

### FreeTube Version

v0.19.1 Beta

### Operating System Version

Windows 11

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by trostboot** [comment]
mpc-hc requires the YDLExePath path in Options > Advanced to be set. Pointing it to yt-dlp instead works just fine. Not an issue with FreeTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Thank you for helping out @trostboot hopfully im not bothering you too much with this

**Comment by trostboot** [comment]
It's fine, I don't mind the pings.

**Comment by 2400YrsOldLallo** [comment]
Thanks for quick update. I just checked mpc-hc Options and YDLExePath is empty. I also checked files in portable.zip and couldn't find yt-dlp file. Could you help me please?

**Comment by 2400YrsOldLallo** [comment]
Nvm, I figured it out. Thanks

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
inquires regarding external players migrated to https://github.com/FreeTubeApp/FreeTube/discussions/7789
